ovs/log/healthcheck_logHandler.py

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/python
+
+# Copyright 2014 iNuron NV
+#
+# Licensed under the Open vStorage Modified Apache License (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.openvstorage.org/license
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+LogHandler module for OVS health check
+"""
+
+import datetime
+import json
+import os
+from ovs.extensions.healthcheck.utils.extension import Utils
+
+
+class _Colors:
```

Comment:
I'd suggest to make this an internal class of ```HCLogHandler```, it makes more sense to have this ```_<class>```.